### PR TITLE
Enable ARM to the Private Link connection Cmdlets

### PR DESCRIPTION
**Pull request by Vishakha-Git**
## Description

Enabling Microsoft.Authorization/resourceManagementPrivateLinks owned by the Azure Resource Manager team to the Private Link Common Cmdlets

Follow-up for Onboarding PR https://github.com/Azure/azure-powershell/pull/15858

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ X] The title of the PR is clear and informative
- [ x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Network/Network/PrivateLinkService/PrivateLinkServiceProvider/ProviderConfiguration.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@ static ProviderConfiguration()
             RegisterConfiguration("Microsoft.ApiManagement/service", "2021-04-01-preview", true, true);
             RegisterConfiguration("Microsoft.AppConfiguration/configurationStores", "2020-06-01", true, true);
             RegisterConfiguration("Microsoft.Attestation/attestationProviders", "2020-10-01", true, true);
-            RegisterConfiguration("Microsoft.Authorization/resourceManagementPrivateLinks", "2020-05-01", false);
+            RegisterConfiguration("Microsoft.Authorization/resourceManagementPrivateLinks", "2020-05-01", true, true);
```
Comment (BethanyZhou) [inline src/Network/Network/PrivateLinkService/PrivateLinkServiceProvider/ProviderConfiguration.cs:20]: Hi @Vishakha-Git , could you pull latest main code into your branch? Current we have three flags to be filled when register configuration.

**Review by BethanyZhou — COMMENTED** [review]
So the cmdlet works even swagger didn't contain the API? @Vishakha-Git

**Comment by Vishakha-Git** [comment]
> So the cmdlet works even swagger didn't contain the API? @Vishakha-Git



> So the cmdlet works even swagger didn't contain the API? @Vishakha-Git

Yes they worked

**Comment by Vishakha-Git** [comment]
> So the cmdlet works even swagger didn't contain the API? @Vishakha-Git



> So the cmdlet works even swagger didn't contain the API? @Vishakha-Git

Yes they worked

**Comment by Vishakha-Git** [comment]
> > So the cmdlet works even swagger didn't contain the API? @Vishakha-Git
> 
> > So the cmdlet works even swagger didn't contain the API? @Vishakha-Git
> 
> Yes they worked

Any suggestion @[BethanyZhou](https://github.com/BethanyZhou) not sure why the test are failing on Windows and MacOS. Also from other PRs enabling providerConfig for private links haven't checked-in any testcase. RegisterConfiguration;

Should I remove the testcases in this PR?

**Comment by BethanyZhou** [comment]
The failure in windows is thrown by Batch Tests and others are caused by timeout @VeryEarly ,  could you have a look?
  [xUnit.net 00:00:03.78]     Microsoft.Azure.Commands.Batch.Test.ScenarioTests.BatchAccountTests.TestGetBatchSupportedImages [FAIL]

@Vishakha-Git ,our test framework does not support testing data-plane. if your feature is in data-plane, please mark it as Category.LiveOnly. If not, I think it's great to have a test.

**Comment by Vishakha-Git** [comment]
> The failure in windows is thrown by Batch Tests and others are caused by timeout @VeryEarly , could you have a look? [xUnit.net 00:00:03.78] Microsoft.Azure.Commands.Batch.Test.ScenarioTests.BatchAccountTests.TestGetBatchSupportedImages [FAIL]
> 
> @Vishakha-Git ,our test framework does not support testing data-plane. if your feature is in data-plane, please mark it as Category.LiveOnly. If not, I think it's great to have a test.

Got it Thanks.

**Comment by Vishakha-Git** [comment]
Are we waiting for a fix for the Batch Test that is timing out?

**Pull request merged**